### PR TITLE
Bug #16564: Red warning when moving .system dataset

### PR DESCRIPTION
**Pull request by neharathod**
In this commit, I have added a warning when user selects a new .system dataset.

### CODE REVIEW

File: gui/freeadmin/static/lib/js/freeadmin.js [hunk 1]
```diff
@@ -1416,6 +1416,19 @@ require([
...
     }
 
+    systemDatasetMigration = function() {
+        var dialog = new Dialog({
+            title: gettext("Warning!"),
+            id: "Warning_box_dialog",
+            content: domConstruct.create(
+                "p", {
+                    innerHTML: gettext(
+                        gettext("Warning: The action will result in migration of dataset.<br />Some services will be restarted.<br /><br />")
```
Comment (william-gr) [inline gui/freeadmin/static/lib/js/freeadmin.js]: I guess you did not understand me.

First, you are using gettext inside a gettedt, which is wrong. Second you are still using html tags within gettext...


Should be... gettext("Warning: The action will result in migration of dataset.") + "<br/>" + gettext(".....");

Comment (neharathod) [inline gui/freeadmin/static/lib/js/freeadmin.js]: Did you mean something like: 'innerHTML: gettext("The action will result in migration of dataset.") + "<br />" + gettext("Some services will be restarted.")'?

Comment (william-gr) [inline gui/freeadmin/static/lib/js/freeadmin.js]: innerHTML: gettext("The action will result in migration of dataset.") + "<br />" + gettext("Some services will be restarted.")

**Comment by william-gr** [comment]
Not quite right, but I will fix myself.

**Pull request merged**